.gitignore

```diff
@@ -8,3 +8,8 @@ www/
 *.log
 *.tmp
 *.tmp.*
+# do not commit macOS specific files
+**/.DS_Store
+# do not commit generated images
+resources/**/icon/
+resources/**/splash/
```

Comment:
See above.

Reply:
Note that removing these files also breaks the Travis build (which doesn't run `ionic cordova resources` on purpose).